src/Form/SponsorshipType.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace AppBundle\Form;
+
+use AppBundle\Entity\ElectedRepresentative\Sponsorship;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class SponsorshipType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('presidentialElectionYear', TextType::class, [
```

Comment:
should be ChoiceType, no?

Reply:
It has no importance, this champ is not modifiable, and `TextType` is more readable.